open_by_laws/views/home.py

```diff
@@ -7,6 +7,16 @@ class HomePageView(LiiWebPageView):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-        codes = "cpt eth jhb wc033 wc011 ec443 wc013 wc012 wc015 wc041 wc023".split()
-        context["municipalities"] = Locality.objects.filter(code__in=codes)
+
+        codes = "cpt eth jhb wc015 wc041 wc023".split()
```

Comment:
This will be stored locally as Locality objects, but the others below won't be, because we don't want them showing up in the database.

So for the ones below, please just put together fake localities like:

```python
[{
  "name": "Bergrivier", "code": "za-wc013", "url": "https://bergrivier.openbylaws.org.za",
  ...
}]